br/pkg/restore/import.go

```diff
@@ -270,32 +303,17 @@ func (importer *FileImporter) Import(
 ) error {
 	start := time.Now()
 	log.Debug("import file", logutil.Files(files))
+
 	// Rewrite the start key and end key of file to scan regions
-	var startKey, endKey []byte
-	if importer.isRawKvMode {
-		startKey = files[0].StartKey
-		endKey = files[0].EndKey
-	} else {
-		for _, f := range files {
-			start, end, err := rewriteFileKeys(f, rewriteRules)
-			if err != nil {
-				return errors.Trace(err)
-			}
-			if len(startKey) == 0 || bytes.Compare(startKey, start) > 0 {
-				startKey = start
-			}
-			if bytes.Compare(endKey, end) < 0 {
-				endKey = end
-			}
-		}
+	startKey, endKey, err := importer.getKeyRangeForFiles(files, rewriteRules)
+	if err != nil {
+		return errors.Trace(err)
 	}
 
-	log.Debug("rewrite file keys",
-		logutil.Files(files),
-		logutil.Key("startKey", startKey),
-		logutil.Key("endKey", endKey))
+	log.Debug("rewrite file keys", logutil.Files(files),
```

Comment:
it's better move this log to https://github.com/pingcap/tidb/pull/32847/files#diff-7b2d077bc48a0da7a3532fa8cdce121bf541ea6146dd8b064bdee212a84a7e55R278